src/makeCalculator.js

```diff
@@ -36,8 +36,36 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
-}
+const makeCalculator = () => {
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
+    },
+    operate(callback, number) {
+      callback(number);
+
+      return calculator;
+    },
+
+    add(number) {
+      calculator.result += number;
```

Comment:
you can refer to the `calculator` as `this` to make it clear that the `result` is taken from the context